README.md

```diff
@@ -56,6 +62,88 @@ If you have a Snyk token, add that as an environment variable:
 - Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables.
 - Set the variable name to _EXHORT_SNYK_TOKEN_, and copy-and-paste your Snyk token into the value field.
 
+
+#### General Configuration
+ Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables, And there you can set the following settings:
+ - name: _EXHORT_DEBUG_, Value: true , Description: Will invoke the analysis in verbose mode and will print a lot of useful logs to job output console - good for debugging, Default value is false.
+
+
+ - name: _EXHORT_DEV_MODE_, value: true, Description: Will invoke the Analysis on Staging Instance Of EXHORT Service, Default: false ( EXHORT Production Instance)
+
+
+ - name: _HIGHEST_ALLOWED_VULN_SEVERITY_, Possible values: [`LOW`,`MEDIUM`,`HIGH`,`CRITICAL`], Description: will determine what is the highest allowed Severity of a vulnerability found for a given package/dependency in the analysis, for the analysis to be considered Successful(RC=0) and not Vulnerable(RC=2), Default value is `MEDIUM`
+
+#### Python Pipeline Configuration
+ For python PIP Packages, you can bypass using the python and pip binaries during the invocation of the analysis, and just use them elsewhere in the pipeline job (another stage, or even another agent/node),
```

Comment:
Python should be capitalized, but not `Packages`.   You need to be consistent with the acronym "PIP", either always capitalize it or don't, although I think it should be capitalized.  Also, we shouldn't use parenthetical statements in technical documentation.  Here is my suggested rewrite:

`For python PIP Packages, you can bypass using the python and pip binaries during the invocation of the analysis, and just use them elsewhere in the pipeline job (another stage, or even another agent/node),` -->

`For Python PIP packages, you can bypass using the Python and PIP binaries during the invocation of the analysis. You can use them elsewhere in your pipeline jobs, such as another stage, or another agent or node.`

Reply:
Done